runtime/near-vm-errors/src/lib.rs

```diff
@@ -3,9 +3,7 @@ use near_rpc_error_macro::RpcError;
 use serde::{Deserialize, Serialize};
 use std::fmt;
 
-#[derive(
-    Debug, Clone, PartialEq, Eq, BorshDeserialize, BorshSerialize, Deserialize, Serialize, RpcError,
-)]
+#[derive(Debug, Clone, PartialEq, Eq, BorshDeserialize, BorshSerialize, Deserialize, Serialize)]
```

Comment:
Why are we removing `RpcError`? Don't we want to generate error schema for it?

Reply:
Yes. It's not exposed outside of the Runtime. Inner fields are converted to either `StorageError` or to `FunctionCallError`